features/bootstrap/FeatureContext.php

```diff
@@ -26,6 +28,30 @@ public function getSession($name = null)
         return $this->getMink()->getSession($name);
     }
 
+    /**
+     * Dump current page data when step failed to allow easy debug on TravisCI.
+     *
+     * @AfterStep
+     */
+    public function dumpPageAfterFailedStep(AfterStepScope $event)
```

Comment:
do we need a place to store screenshot? I can provision a public S3 bucket for that.

Reply:
> 
> 
> On the PR - it only adds the method. At least some documentation is necessary on how others can use it.

It has on step hook, so it is added automatically activated for every failed step.

Reply:
> 
> 
> do we need a place to store screenshot? I can provision a public S3 bucket for that.

I was planning to use imgur.com, but S3 is good - can you please add it directly to this PR? And please - do not forget to handle automatic expires after some time + always put there a link to delete - important, when some sensitive data will be dumpled/saved by a mistake.

S3 can be also used for dumping the page source.